qutebrowser/utils/utils.py

```diff
@@ -856,3 +856,18 @@ def expand_windows_drive(path):
         return path + "\\"
     else:
         return path
+
+def is_mac():
+    if sys.platform.startswith('darwin'):
```

Comment:
Sorry for nitpicking this, but can't these functions be:

```
def is_mac():
    return sys.platform.startswith('darwin'):
```

:P

Reply:
Just without the colon in the end :P 

Done and done.